program-other-programmes.html

```diff
@@ -0,0 +1,215 @@
+---
+layout: default
+title: Other Programmes
+body_class: page page-programme page-other-programmes
+---
+
+<div class="container">
+    <div class="row">
+        <div class="content-constrained">
+            <section id="writing-club">
+                <h2>Writing Club</h2>
+                <p>
+                    Are you passionate about blogging, eager to share your ideas, and
+                    looking to improve your writing skills?
+                </p>
+                <p>
+                    The Online Writing Club is your perfect destination to practise blog
+                    posting, collaborate with fellow writers, and refine your craft—all
+                    from the comfort of your home.
+                </p>
+                <p>
+                    Whether you’re a seasoned blogger, an aspiring writer, or simply
+                    someone who loves to share stories and ideas, our club offers a
+                    supportive and stimulating environment for everyone.
+                </p>
+                <div class="faq my-4">
+                    <div class="accordion" id="accordion-1">
+                        <a class="accordion-header" aria-controls="programme-1" aria-expanded="false"
+                            data-toggle="collapse" href="#programme-1" role="button">
+                            <h2 class="accordion-title">What we offer</h2>
+                            <div class="accordion-icon">{% include icons/plus.svg %}</div>
+                        </a>
+                        <div class="accordion-body collapse" id="programme-1">
+                            <ul>
+                                <li>
+                                    <strong>Monthly Blog Writing Sessions:</strong> Join our live,
```

Comment:
Please add a screenshot of the accordion expanded to show the list as well.
Also for the other clubs